signingscript/src/signingscript/sign.py

```diff
@@ -834,6 +835,56 @@ def write_signing_req_to_disk(fp, signing_req):
     fp.write(b"}]")
 
 
+def _encode_multiple_files(fp, signing_req):
```

Comment:
If you've figured these functions out, it might be helpful for the next person if you add a docstring here (and possibly flesh out the docstring above in `_encode_single_file`) describing what it does. (The `why` might be super helpful too, but I'm not sure if we understand that yet?)

Reply:
I added a doc string